.fluence/aqua/deals.aqua

```diff
@@ -0,0 +1,17 @@
+aqua Deals declares *
```

Comment:
added accidentally

do we need these files? why don't they in gitignore

Reply:
they're not in git ignore to share deployed deal info

you deploy, I can use. and vice versa.